<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Admin-PC" name="Application Test" tests="3" failures="1" timestamp="20 Jun 2017 09:56:20 GMT" time="0.015" errors="0">
  <testcase name="doLogin" time="0.002" classname="testcases.ApplicationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error Message">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Error Message
	at testcases.ApplicationTest.doLogin(ApplicationTest.java:51)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="doPasswordChange" time="0.0" classname="testcases.ApplicationTest">
    <skipped/>
  </testcase> <!-- doPasswordChange -->
  <testcase name="doLogout" time="0.0" classname="testcases.ApplicationTest">
    <skipped/>
  </testcase> <!-- doLogout -->
</testsuite> <!-- Application Test -->
